### ISSUE TITLE
[de] Getting rid of nonsensical spell-check suggestions (compounds)

### ISSUE THREAD

**Issue by janschreiber** [languagetool-org/languagetool#753]
Now that we have a [huge number of good suggestions](https://github.com/languagetool-org/languagetool/commit/fd6c167b6f3e297b3b2c91119f2246561e826af6) to choose from, I think it is a good moment to lean back and think about how we can further improve the suggestions.

When I go through [suggestions.txt](https://github.com/languagetool-org/languagetool/blob/master/languagetool-language-modules/de/src/test/resources/suggestions.txt) with a spell checker, I see some things that I really wish weren't there.

Some words never seem to be used in compounds that make sense. Here is a round-up of some of the worst offenders and some silly compounds in which they occur:

- Laos: Tiefkuhlaos
- Orang: Erdorang
- Gen: Urangen, Spülgen
- Luv: Weltluv
- Mars: Weihnachtsmars

Some remarks: 

- AFAIK, 'Orang' occurs only as part of 'Orang-Utan' and should not be used for compounds at all.
- 'Laos' and 'Mars' are proper names and should probably never be used at the end of a compound. 'Westlaos' etc. make sense, but those should be in my list already.
- 'Luv' is a very specialized nautical word. I doubt that anyone would ever want to use a compound of it except 'Luvseite'. I don't think it would do any harm if it isn't considered for compounds any more.

Any other ideas?

**Comment by danielnaber** [comment]
I agree - but I guess this needs to be fixed in the hunspell dictionary? So we'll need to contact its author and wait for a new release (which can take a year or so). Alternatively we'd need to modify the dictionary, but then we'd need to merge our changes with the next update.
